tst/teststandard/bugfix.tst

```diff
@@ -1084,11 +1086,11 @@ gap> testG :=
 >       return (Group(M1));
 >    end;;
 gap> StructureDescription(testG(8,2));
-"(C8 x C4) : C2"
+"((C8 x C2) : C2) : C2"
 gap> StructureDescription(testG(8,3));
 "C3 x QD16"
 gap> StructureDescription(testG(8,4));
-"(C16 x C4) : C2"
+"((C16 x C2) : C2) : C2"
```

Comment:
Ahh, too bad, the old ones looked "nicer" to me... but ignore me, it is probably impossible to making a "perfect" choice here (even if one assumed that all mathematicians had the same "taste", which I am sure we don't :-). Short of hard coding lots of cases, that is.